src/modules/form/hooks/useFormDialog.tsx

```diff
@@ -152,9 +152,14 @@ export function useFormDialog<T extends SubmitFormAllowedTypes>({
             {definitionLoading ? (
               <Loading />
             ) : formDefinition ? (
-              <Grid container spacing={2} sx={{ mb: 2, mt: 0 }}>
+              <Grid
```

Comment:
I _think_ changing this grid's `direction` prop should be all right, because in the previous version of the code it has only one grid item within it, and the child `DynamicForm` component renders its own grid internally. But I know this hook is used in lots more contexts than just the one I'm looking at, so let me know if you think the change has a possibility of unintended side effects, and I can revert this change and instead add a `Stack` that separates the pre form component from the form itself.

Reply:
Yeah this makes me a little nervous just because it's used so widely. Can we just add margin below the preform component instead? Eg `mb` on the `ClientAlertStack` component

Reply:
Yep this looks good to me (deleted my previous comment). The only issue I am seeing is that there is additional padding added when there is no preformcomponent.